src/utils.ts

```diff
@@ -18,6 +18,11 @@ export const toBase64 = (bytes: Uint8Array): string =>
 export const fromBase64 = (str: string): Uint8Array =>
   Buffer.from(str, 'base64');
 
+export const toUtf8 = (bytes: Uint8Array): string =>
+  Buffer.from(bytes).toString('utf-8');
+
+export const fromUtf8 = (str: string): Uint8Array => Buffer.from(str, 'utf-8');
```

Comment:
At lines 1 & 4 you can see similar logic implemented using [Web API's `TextEncoder`](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder). We're primarily targeting browser clients so we try to use it when possible, instead of relying on node.js's `Buffer`.

Reply:
Ah yeah totally skimmed past those first two functions. Implemented suggestions and removed my added functions.